templates/partners/desktop.html

```diff
@@ -0,0 +1,196 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-desktop">
```

Comment:
Can you use just one class for this and add `color: #fff` to the suru styling?

For example:
``` html
<section class="p-strip--suru-desktop">
  <div class="row">
    ...
  </div>
</section>
```